app/controllers/api/users.php

```diff
@@ -591,6 +591,10 @@
         /** @var Utopia\Database\Database $dbForInternal */
         /** @var Appwrite\Stats\Stats $usage */
 
+        if(\strlen($prefs) > 16384) {
```

Comment:
This limit should be a part of a validator for better consistency. We also discussed mentioning the size limit as part of the param description. I think the extension can be located at `src/Appwrite/Auth/Validator/Prefs` unless someone has a better idea. 

> Note:  when adding a new param we need to make sure it is translated to the proper format for the Swagger / OpenAPI exports.